cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ContactDetailsMapper.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import uk.gov.hmcts.cmc.ccd.domain.ContactDetails;
+
+public class ContactDetailsMapper {
+
+    ContactDetails toCCD(uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails contactDetails) {
+        ContactDetails ccdContactDetails = ContactDetails.builder()
+            .email(contactDetails.getEmail().orElse(null))
+            .phone(contactDetails.getPhone().orElse(null))
+            .dxAddress(contactDetails.getDxAddress().orElse(null))
+            .build();
+
+        return ccdContactDetails;
+    }
+
+    uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails toCMC(ContactDetails contactDetails) {
+        uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails cmcContactDetails =
+            new uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "cmcContactDetails". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1488)